### PR TITLE
[chore](scan) Add session veriable query_max_scan_bytes

### PR DESCRIPTION
**Pull request by xinyiZzz**
## Proposed changes

add Query largest scan bytes in single BE.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xinyiZzz** [comment]
run buildall

File: be/src/vec/exec/scan/vscanner.cpp [hunk 1]
```diff
@@ -96,6 +96,7 @@ Status VScanner::get_block(RuntimeState* state, Block* block, bool* eof) {
                 }
                 _num_rows_read += block->rows();
                 _num_byte_read += block->allocated_bytes();
+                RETURN_IF_ERROR(state->get_query_ctx()->add_scan_bytes(block->allocated_bytes()));
```
Comment (yiguolei) [inline be/src/vec/exec/scan/vscanner.cpp:99]: 这里，每10次，调用一下，我怕太频繁了

**Comment by xinyiZzz** [comment]
run buildall

**Comment by xinyiZzz** [comment]
run buildall

**Pull request closed without merging**